inspirehep/modules/crossref/ext.py

```diff
@@ -91,57 +82,4 @@ def init_app(self, app):
         """Initialize a Flask application."""
         app.config.setdefault("CROSSREF_API_URL",
                               "http://api.crossref.org/works/")
```

Comment:
If you use the `init_config` pattern described below you can move this in `config.py`.